src/i18n/i18n.service.ts

```diff
@@ -87,6 +88,8 @@ export class Overridable {
 	 * Our base non-overridden translation.
 	 */
 	protected baseTranslation: Observable<string> = this.i18n.get(this.path);
+
+	protected subscription: Subscription;
```

Comment:
Could you add a one line comment explaining the purposes of this property?

Reply:
Two lines it is

Reply:
Whatever works 😆

Reply:
Also, maybe something like 
```suggestion
	/**
	 * Subscription to the observable provided as an override (if any)
	 */
	protected subscription: Subscription;
```
?

Reply:
This seems to still need done?